java/code/src/com/suse/manager/webui/controllers/SystemsController.java

```diff
@@ -212,7 +213,9 @@ public String delete(Request request, Response response, User user) {
 
         try {
             // Now we can remove the system
-            SystemManager.deleteServer(user, sid);
+            SystemManager systemManager = new SystemManager(ServerFactory.SINGLETON, ServerGroupFactory.SINGLETON,
```

Comment:
should this manager be a class field or do we need to always create a new instance?